packages/terra-table/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "terra-table",
+  "main": "lib/index.js",
+  "version": "4.36.0",
```

Comment:
I thought we were doing a major version bump with this update.

Reply:
This will be automatically updated once we do the release.